ledger/src/blockstore_db.rs

```diff
@@ -1227,6 +1245,39 @@ impl TypedColumn for columns::OptimisticSlots {
     type Type = blockstore_meta::OptimisticSlotMetaVersioned;
 }
 
+impl Column for columns::MerkleRoot {
```

Comment:
This is a copypasta from `ShredCode`, `ShredData`, and `ErasureMeta`. Happy to consolidate but am unsure about the effects of 
```   
 fn as_index(slot: Slot) -> Self::Index {
        (slot, 0)
    }
```

Should I be storing a `fec_set_index` and use it instead of `0` here?

Reply:
What you have for `as_index()` is fine; see this comment in the trail declaration:
https://github.com/solana-labs/solana/blob/56ccffdaa5394f179dce6c0383918e571aca8bff/ledger/src/blockstore_db.rs#L710-L713

And yeah, looks like you might be able to reuse some impl's